src/Components/Publishing/Icon/IconVideoFullscreen.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react"
+
+export class IconVideoFullscreen extends Component<any, null> {
```

Comment:
I _think_ this could be different in reaction (as compared to emission), but if state isn't being used `null` can be omitted

Reply:
Yeah I should actually just change all the icons to be SFC. They're easier to export/style outside of the component that way too. Like with: https://github.com/artsy/reaction/pull/408/files#diff-f5accae7b0f443e61496b5c152e113aaR1

Allows you to do:
```
import { IconVideoPause } from "../../Icons/IconVideoPause"

const IconContainer = styled.div`
   ${IconVideoPause} {
      margin: 20px;
   }
`
```

Reply:
👍